yt_dlp/extractor/youtube/_base.py

```diff
@@ -360,6 +360,7 @@ class YoutubeBaseInfoExtractor(InfoExtractor):
         r'(?:www\.)?invidious\.moomoo\.de',
         r'(?:www\.)?ytb\.trom\.tf',
         r'(?:www\.)?yt\.cyberhost\.uk',
+        r'(?:www\.)?inv\.nadeko\.net',
```

Comment:
can you move this into the previous section (under the `# Invidious instances taken from https://github.com/iv-org/documentation/blob/master/docs/instances.md` heading)